jenkinsfile-runner-base-image/packager-config.yml

```diff
@@ -483,7 +483,7 @@ plugins:
   - groupId: "io.jenkins.plugins"
     artifactId: "warnings-ng"
     source:
-      version: "8.3.0"
+      version: "8.4.4"
```

Comment:
@srinikitha09 Shall we give warnings-ng 8.4.4 a try? It would make our lives easier if we would not need to stick to an old version of this plugin all the time.

Reply:
Sure, but I doubt that it may not work with our current cloud sdk version `v39 ` but probably with higher versions.

Reply:
I remember I tested pipelines and we had issues

Reply:
I commented already in the merged PR: https://github.com/SAP/stewardci-jenkinsfilerunner-image/pull/57#issue-495463183

Reply:
also here: https://github.com/SAP/stewardci-jenkinsfilerunner-image/pull/49#discussion_r497381008

Reply:
downgraded

Reply:
I'll get the version from https://gkecloudciservice.jaas-gcp.cloud.sap.corp/job/steward-ci-os/job/stewardci-jenkinsfilerunner-image-pipeline/214 and test it on perf2 cluster